Essentials/src/messages.properties

```diff
@@ -2,144 +2,144 @@
 #version: ${full.version}
 # Single quotes have to be doubled: ''
 # by:
-action=\u00a75* {0} \u00a75{1}
-addedToAccount=\u00a7a{0} has been added to your account.
-addedToOthersAccount=\u00a7a{0} added to {1}\u00a7a account. New balance\: {2}
+action=\u00A75* {0} \u00A75{1}
+addedToAccount=\u00A7a{0} has been added to your account.
+addedToOthersAccount=\u00A7a{0} added to {1}\u00A7a account. New balance\: {2}
```

Comment:
This whole file is full of changes like this. I don't see any reason behind this

Reply:
Oh my, that's my IDE settings with native-to-ascii conversion. Fixed!